design/design-file_organization.md

```diff
@@ -0,0 +1,93 @@
+
+
+Within the **UI Design** (and **Brand Design**) folders, files are named with a consistent pattern so that it is easy to know which is most current while still maintaining a project’s history. A new version of a file is added whenever there is a milestone in the project. For example, if the design direction changes, after a demo, etc.
+
+The goal is to no longer use an “Archive” tab within Sketch and instead use files to show the archived state. This allows us to use clean symbols and have smaller working files while still keeping a historical view of the project.
+
+Whoever is working on a particular file adds their initials at the end so that multiple designers can be working on the same project without overwriting each other.
+
+There is never a “final” file. Instead, the most recent numbered file is the most current file.
+
+
+## Root / -Projects / Project Name / -UI Iterations
+
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_76A2CDE662BC80FFEE513EA72F067BE0FAEA76B09B91F12E302C16DEA62A7533_1536345369339_Screen+Shot+2018-09-07+at+2.35.07+PM.png)
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_76A2CDE662BC80FFEE513EA72F067BE0FAEA76B09B91F12E302C16DEA62A7533_1536345369332_Screen+Shot+2018-09-07+at+2.35.45+PM.png)
+
+
+There are two ways the **-UI Iterations** folder could be structured. Every **-UI Iterations** folder will contain a **Symbol Library** file, which will serve as the source of truth for all symbols and consistent styles across the project. It can either have a list of versioned files that use that symbol library, or it can have subfolders like **-Foundry** or **-Newsletter** that represent specific parts of a larger project. These sub-folders should be used when the project is too large to contain in one Sketch file or to make it easier for multiple designers to work at the same time.
+
+
+## Root / Internal
+
+This folder holds all relevant assets that any Sparkbox designer might need access to or find helpful as they work on projects. Current internal projects live in this folder as well and are indexed with a **-**. Here’s an example of how this folder could be structured:
+
+- **Foundry Images**
+- **Brand**
+- **Bookshelf`
+- **Dribbble**
+- **Photos**
+- **Resources**
+- **Swag and Decor**
+
+
+Root / Internal / -Foundry Images
+- **Exported Final Images** includes all the exports of approved Foundry images in their compressed format
+- **Working Images** includes all the active art files
+- **In Article Images** includes in-article images for posts that contain those. Inside this folder is a folder for each post that is named in the same format as a Foundry image (see below).
+- **Video Graphics** includes in-article images for posts that contain those. Inside this folder is a folder for each post that is named in the same format as a Foundry image (see below).
```

Comment:
Looks like this line was just copied from In Article Images. What should this say?